dpl-docs/views/layout.dt

```diff
@@ -113,6 +113,8 @@ html(lang='en-US')
         h1 <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[0 .. $-24]}</code> #{title[$-24 ..$]}
       - else if (hidx > 0 && title[0 .. hidx] != "Module" && info.mod)
         h1 #{title[0 .. hidx]} <code><a href="#{info.linkTo(info.mod)}">#{info.mod.qualifiedName}</a>.#{title[hidx+1 .. $]}</code>
+      - else if (hidx > 0 && title[0 .. hidx] == "Module" && info.mod)
+        h1 Module <code>#{info.mod.qualifiedName}</code>
```

Comment:
Thanks for looking into this! Could we expand ddoc macros here instead of hardcoding tags? I'm thinking e.g. $(D #{info.mod.qualifiedName}).